locations/spiders/jax_tyres_and_auto_au.py

```diff
@@ -37,7 +37,7 @@ def parse(self, response):
         day_names = response.xpath('//div[contains(@class, "pgStLcSn-openingHours")]/div[1]/p/text()').getall()
         hour_ranges = response.xpath('//div[contains(@class, "pgStLcSn-openingHours")]/div[2]/p/text()').getall()
         for i in range(1, 7, 1):
-            if hour_ranges[i].upper() == "CLOSED":
+            if not any(char.isdigit() for char in hour_ranges[i]):
                 continue
             open_time = hour_ranges[i].split(" - ")[0]
             close_time = hour_ranges[i].split(" - ")[1]
```

Comment:
This would also crash if another holiday used a number in there? eg "Good Friday 7th" Or if the times weren't padded by a space, just "07:45 AM-05:00 PM".

It'd be safer to use regex here:
```suggestion
            if m := re.match(r"(\d\d:\d\d [AP]M)\s*-\s*(\d\d:\d\d [AP]M)", hour_ranges[i]):
                properties["opening_hours"].add_range(day_names[i], m.group(1), m.group(2), "%I:%M %p")
```

Reply:
Thanks for the suggestion, that's a much better approach. I've fixed it :)